envoy/http/alternate_protocols_cache.h

```diff
@@ -99,6 +99,13 @@ class AlternateProtocolsCache {
   virtual void setAlternatives(const Origin& origin,
                                std::vector<AlternateProtocol>& protocols) PURE;
 
+  /**
+   * Sets the srtt estimate for an origin, assuming the origin exists in the cache.
```

Comment:
Maybe include what happens if it's not in the cache? Seems like we just noop, but I wasn't sure until I checked the code